include/variant_parser/variant_record.hpp

```diff
@@ -66,16 +66,21 @@ class variant_header
      *
      * \tparam stream_type - a stream to print the output to
      *
+     * \param[in, out] sn_ln_tag_tuple_set - a vector of \@SQ header tags tuples: SN and LN
      * \param[in, out] out_stream - the output stream to print to
      */
     template<typename stream_type>
     //!cond
         requires seqan3::output_stream<stream_type>
     //!endcond
-    void print(stream_type & out_stream)
+    void print(std::set<std::tuple<std::string, int32_t>> & sn_ln_tag_tuple_set, stream_type & out_stream)
     {
         out_stream << "##fileformat=" << fileformat << '\n';
         out_stream << "##source=" << source << '\n';
+
+        for(std::tuple<std::string, int32_t> contig : sn_ln_tag_tuple_set)
+            out_stream << "##contig=<ID=" << std::get<0>(contig) << ",length=" << std::get<1>(contig) << ">\n";
+
```

Comment:
Structure bindings also work in this context. I think it makes the code more readable. It also avoids an unnecessary std::string copy, which depending on the length of the id doesn't really matter.
```suggestion
        for(auto const& [id, length] : sn_ln_tag_tuple_set)
            out_stream << "##contig=<ID=" << id << ",length=" << length << ">\n";

```